recipe/meta.yaml

```diff
@@ -14,24 +14,24 @@ source:
 build:
   number: 0
   script:
-    - python setup.py build_ext --compiler=mingw32  # [win and py2k]
+    - python setup.py build_ext --compiler=mingw32 -I%LIBRARY_INC%  # [win and py2k]
```

Comment:
This should have been `-I%LIBRARY_PREFIX%\mingw-w64\include`

Reply:
Think we can just drop that flag as it already looks there.

Reply:
Hmm...apparently this search is broken as demonstrated by [this build]( https://ci.appveyor.com/project/conda-forge/backports-lzma-feedstock/build/1.0.48/job/tk6wjbnknu73wamh ). Have included a fix for this and the library search path in a recent commit.

@nehaljwani @peterjc, any thoughts as to why it doesn't look there?